membership_policy.md

```diff
@@ -128,6 +130,45 @@ The following apply to the part of project for which one would be an approver in
   * Mentor contributors and reviewers
   * May approve contributions for acceptance
 
+## SIG Chair
+
+### Requirements
+
+* contributor
+* advanced knowledge in SIG subject
+
+### Responsibilities and privileges
+
+* is an organizer and facilitator
+* operates the SIG
+* communicates and coordinates with other SIGs, and the broader community
+* may claim approver rights in scope
+
+## SIG Subproject Lead
+
+### Requirements
+
+* contributor
+* advanced knowledge in subproject scope
+
+### Responsibilities and privileges
+
+* leads within subproject scope
+* is an active reviewer
+* may claim approver rights in scope
+
+## WG Chair
+
+### Requirements
+
+* contributor
+* advanced knowledge in WG scope
+
+### Responsibilities and privileges
+
+* is an organizer and facilitator
+* gives updates to respective sponsoring SIG Chairs
```

Comment:
I'd want all these points to start with upper case ;)

Reply:
BUT WHY?

Reply:
Kidding - can do